Misc/NEWS.d/next/Core_and_Builtins/2025-04-06-17-31-44.gh-issue-132166.tbo9yR.rst

```diff
@@ -0,0 +1,5 @@
+Add checking if sequences ``a[alo:ahi]`` and ``b[blo:bhi]`` are the same on the
+beginning of the method find_longest_match in SequenceMatcher. For identical
+sequences there is no reason to run whole logic when simple check can be done.
```

Comment:
This NEWS should be summarized. I suggest something like (assumiong that the method is documented, if not use a plain

```rst
Add a fast path in :meth:`SequenceMatcher.find_longest_match
<difflib.SequenceMatcher.find_longest_match>` to handle the
case when ``a[alo:ahi] == b[blo:bhi]``.
```